src/openrct2/management/NewsItem.cpp

```diff
@@ -260,28 +256,24 @@ std::optional<CoordsXYZ> news_item_get_subject_location(int32_t type, int32_t su
             }
 
             // Find the first car of the train peep is on
-            sprite = try_get_sprite(ride->vehicles[peep->CurrentTrain]);
+            auto sprite = TryGetEntity<Vehicle>(ride->vehicles[peep->CurrentTrain]);
             // Find the actual car peep is on
             for (int32_t i = 0; i < peep->CurrentCar && sprite != nullptr; i++)
             {
-                sprite = try_get_sprite(sprite->vehicle.next_vehicle_on_train);
+                sprite = TryGetEntity<Vehicle>(sprite->next_vehicle_on_train);
             }
-            if (sprite != nullptr)
+            if (sprite)
```

Comment:
```suggestion
            if (sprite != nullptr)
```
Just because that's how we do it elsewhere (I personally would rather not have the `== nullptr` or `!= nullptr`)